scripts/smart-table.js

```diff
@@ -92,6 +96,8 @@ $(function () {
           pillContainer.prepend($pill[0]);
           pillFilters.add(cellValue);
         }
+
+        smartTableCallbacks.dataFiltered.forEach(cb => cb(filters));
```

Comment:
Using a list of callbacks here is really clever! Gives us a nice way to expand this in the future if the need arises.